src/wpt/fs-test.ts

```diff
@@ -69,38 +69,22 @@ export default {
   },
   'FileSystemDirectoryHandle-getDirectoryHandle.https.any.js': {
     comment: '...',
-    expectedFailures: [
-      // TODO(node-fs): Fix these tests
-      'getDirectoryHandle(create=false) with a path separator when the directory exists',
-      'getDirectoryHandle(create=true) with a path separator',
-    ],
+    expectedFailures: [],
```

Comment:
We can just remove all contents inside the parent object

Reply:
Yep, this is temporary while I'm working through these... I'm using it as a visual reminder that I've already looked at these. It'll all end up with good formatting when all done.